luajit.krun

```diff
@@ -46,6 +46,11 @@ VMS = {
         'variants': ['default-lua'],
         'n_iterations': ITERATIONS_ALL_VMS,
     },
+    'RaptorJIT': {
+        'vm_def': LuaVMDef("builds/raptorjit", env = VM_ENV),
```

Comment:
In Python, typically kwargs (keyword args) don't use spaces around `=`. I know, it's inconsistent, but that's the way the Python folk do it :P

Reply:
fixed